# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/DeleteTextTransaction.cpp
# Commit: d3816f82c0b9
# Full Hash: d3816f82c0b9d4ceca0bb2ad177694164ed0921f
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2020-04-03 21:40:10
# Regressor Bug: 1619914
# File Overlap Count: 1
# Description:
#   Bug 1619914 - part 2: Mark transaction class methods and their caller methods as `MOZ_CAN_RUN_SCRIPT r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D69303
# ==============================================================================

diff -r 876b1031a6ae -r d3816f82c0b9 editor/libeditor/DeleteTextTransaction.cpp
--- a/editor/libeditor/DeleteTextTransaction.cpp	Fri Apr 03 01:38:22 2020 +0000
+++ b/editor/libeditor/DeleteTextTransaction.cpp	Fri Apr 03 01:39:36 2020 +0000
@@ -97,8 +97,7 @@
   return mEditorBase->IsModifiableNode(*mTextNode);
 }
 
-MOZ_CAN_RUN_SCRIPT_BOUNDARY NS_IMETHODIMP
-DeleteTextTransaction::DoTransaction() {
+NS_IMETHODIMP DeleteTextTransaction::DoTransaction() {
   if (NS_WARN_IF(!CanDoIt())) {
     return NS_ERROR_NOT_AVAILABLE;
   }
@@ -140,8 +139,7 @@
 
 // XXX: We may want to store the selection state and restore it properly.  Was
 //     it an insertion point or an extended selection?
-MOZ_CAN_RUN_SCRIPT_BOUNDARY NS_IMETHODIMP
-DeleteTextTransaction::UndoTransaction() {
+NS_IMETHODIMP DeleteTextTransaction::UndoTransaction() {
   if (NS_WARN_IF(!CanDoIt())) {
     return NS_ERROR_NOT_AVAILABLE;
   }